common-custom-user-data-maven-extension/src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -40,14 +40,16 @@ private static void captureOs(BuildScanApi buildScan) {
     }
 
     private static void captureIde(BuildScanApi buildScan) {
-        if (sysProperty("idea.version").isPresent()) {
-            buildScan.tag("IntelliJ IDEA");
-            buildScan.value("IntelliJ IDEA version", sysProperty("idea.version").get());
-        } else if (sysProperty("eclipse.buildId").isPresent()) {
-            buildScan.tag("Eclipse");
-            buildScan.value("Eclipse version", sysProperty("eclipse.buildId").get());
-        } else if (!isCi()) {
-            buildScan.tag("Cmd Line");
+        if (!isCi()) {
+            if (sysProperty("idea.version").isPresent()) {
+                buildScan.tag("IntelliJ IDEA");
+                buildScan.value("IntelliJ IDEA version", sysProperty("idea.version").get());
+            } else if (sysProperty("eclipse.buildId").isPresent()) {
+                buildScan.tag("Eclipse");
+                buildScan.value("Eclipse version", sysProperty("eclipse.buildId").get());
+            } else if (!isCi()) {
```

Comment:
@jthurne this check is now redundant. (see the gradle plugin for comparison)

Reply:
Once fixed, you can merge to master.

Reply:
> @jthurne this check is now redundant.

It is indeed! Fixed.